tests/pypi/extension/pip_parse.bzl

```diff
@@ -41,7 +42,9 @@ def pip_parse(
         envsubst = envsubst,
         experimental_index_url = experimental_index_url,
         experimental_requirement_cycles = experimental_requirement_cycles,
+        # TODO @aignas 2025-12-02: decide on a single attr - should we reuse this?
```

Comment:
What's the difference in behavior? Both are a list of platform names (as defined by the pip.platforms() configuration). Is it just that experimental_target_platforms has the `*` matching, and this one doesn't?